fsharp-backend/src/LibAnalysis/LibAnalysis.fs

```diff
@@ -94,23 +93,26 @@ module Eval =
       return results
     }
 
-module CT = ClientTypes
-module CTA = ClientTypes.Analysis
+module CTRuntime = ClientTypes.Runtime
+module CTAnalysis = ClientTypes.Analysis
+module CT2Program = ClientTypes2ExecutionTypes.ProgramTypes
 
-let performAnalysis (args : CTA.PerformAnalysisParams) : Task<CTA.AnalysisEnvelope> =
+let performAnalysis
+  (args : CTAnalysis.PerformAnalysisParams)
+  : Task<CTAnalysis.AnalysisEnvelope> =
   let runAnalysis
     (tlid : tlid)
-    (traceID : CTA.TraceID)
-    (traceData : CTA.TraceData.T)
+    (traceID : CTAnalysis.TraceID)
+    (traceData : CTAnalysis.TraceData)
```

Comment:
It seems odd that these take clients types for some arguments. I think if we're going to convert from clients types in one place, we should do the conversion for all the arguments in that place.

Reply:
I agree. I ended up extracting this mapping out to `CT2Analysis`, including the bits where we map from PTs to RTs. I think this turned out much nicer.